roles/tendrl-server/README.md

```diff
@@ -74,6 +74,33 @@ Role Variables
     development/testing purposes only. For production, keep the authentication
     always enabled.
 
+ *  When both variables `tendrl_notifier_email_id` and
+    `tendrl_notifier_email_smtp_server` are left undefined (which is the
+    default state), email configuration of tendrl-notifier is skipped.
```

Comment:
I would somehow rephrase this paragraph, because it sounds like email configuration is skipped only when *both* mentioned variables are undefined, but in reality email configuration is skipped even if one of the variables is undefined - which is quite clear from the rest of the file.

Reply:
Ah, good point.